saleor/giftcard/utils.py

```diff
@@ -61,6 +61,8 @@ def remove_gift_card_code_from_checkout(checkout: Checkout, gift_card_code: str)
     if gift_card:
         checkout.gift_cards.remove(gift_card)
         checkout.save(update_fields=["last_change"])
+        return True
+    return False
```

Comment:
I will suggest also extending the docs with the same annotation as it was done in the function for removing the voucher:
`Return information whether promo code was removed.`